client/src/components/core/ms-input/MsPasswordInput.vue

```diff
@@ -1,48 +1,55 @@
 <!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
 
 <template>
-  <ion-grid class="input-grid">
-    <ion-row>
-      <ion-col class="input-container">
-        <ion-text
-          id="passwordLabel"
-          class="form-label"
-        >
-          {{ label }}
-        </ion-text>
-        <ion-item class="input-item ion-no-padding">
-          <ion-input
-            class="form-input input"
-            ref="inputRef"
-            aria-labelledby="passwordLabel"
-            :type="passwordVisible ? 'text' : 'password'"
-            @ion-input="onChange($event.target.value)"
-            :value="modelValue"
-            @keyup.enter="onEnterPress()"
-            id="ms-password-input"
-            :clear-on-edit="false"
-            @ion-focus="hasFocus = true"
-            @ion-blur="hasFocus = false"
-          />
-          <div
-            class="input-icon"
-            @click="passwordVisible = !passwordVisible"
-          >
-            <ion-icon
-              slot="icon-only"
-              :icon="passwordVisible ? eyeOff : eye"
-            />
-          </div>
-        </ion-item>
-      </ion-col>
-    </ion-row>
-  </ion-grid>
+  <div class="input-container">
+    <span
+      id="passwordLabel"
+      class="form-label"
+    >
+      {{ label }}
+    </span>
+    <ion-item
+      class="input-item"
+      :class="isWrong ? 'input-invalid' : ''"
+    >
+      <ion-input
+        class="form-input input"
+        ref="inputRef"
+        aria-labelledby="passwordLabel"
+        :type="passwordVisible ? 'text' : 'password'"
+        @ion-input="onChange($event.target.value)"
+        :value="modelValue"
+        id="ms-password-input"
+        :clear-on-edit="false"
+        @keyup.enter="onEnterPress()"
```

Comment:
You've removed `@ion-focus` and `@ion-blur`. Probably a bad conflict resolution, but be careful.

Reply:
Yes, because of rebase I think. I added it back.